src/terrain/routes/filesystem.clj

```diff
@@ -27,7 +27,10 @@
 (defn- fix-sort-col-param [param-value]
   (as-> param-value v
     (string/lower-case v)
-    (if (= v "lastmodified") "datemodified" v)))
+    (cond
+      (= v "id")           "path"
+      (= v "lastmodified") "datemodified"
+      :else                v)))
```

Comment:
I know this is already merged but being slightly picky, I'd prefer `condp` here:

```clojure
(condp = v
    "id" "path"
    "lastmodified" "datemodified"
    v)
```

cuts down a little on the repetition of `(= v ...)`

Reply:
I have mixed feelings about `condp`, which was the primary reason I didn't use it. I like to have all of the conditions (or condition arguments) aligned on the left and all of the actions (or return values) aligned on the right. My editor will do that for me when I use `cond`, but not when I use `condp`. 😆 I haven't tried to figure out how to get Clojure mode in Emacs to align `condp` the way I like yet, so I generally stick with `cond`. That's not the greatest reason to do so, but it's the only reason I have at the moment. 🤣